R/exec_profile.R

```diff
@@ -0,0 +1,36 @@
+exec_profile <- function(.fn, indicator, indicator_after) {
+  tilt_indicator_output <- exec(get(.fn), !!!indicator)
+
+  product <- unnest_product(tilt_indicator_output)
+  if (grepl("emissions", .fn)) {
+    co2 <- indicator[[2]]
+    product <- extend_product(product, co2)
+  }
+  company <- unnest_company(tilt_indicator_output)
```

Comment:
@maurolepore This logic only adds extra columns to Emission profile indicators. Please make sure that`isic_4digit` column will be present in the final output of sector_profile indicators. That's what I want to be sure about using this issue: https://github.com/2DegreesInvesting/tiltIndicator/issues/631

Reply:
Done